### ISSUE TITLE
Test name prefix "Exam Partition #XX - " is missing on testem >= 1.16.0

### ISSUE THREAD

**Issue by SergeAstapov** [trentmwillis/ember-exam#88]
Steps to reproduce:
`ember new ember-quickstart && cd ember-quickstart`

Update testem.js and add "parallel": -1 option.

Not necessary, but just to have some more tests:
`ember generate route users && ember generate acceptance-test users`

Command `ember install ember-exam && ember exam --split 3 --parallel` produces following output:
```
ok 1 PhantomJS 2.1 - Acceptance | users: visiting /users
ok 2 PhantomJS 2.1 - ESLint | app: app.js
ok 3 PhantomJS 2.1 - ESLint | app: resolver.js
ok 4 PhantomJS 2.1 - ESLint | app: router.js
ok 5 PhantomJS 2.1 - ESLint | app: routes/users.js
...
```

After downgrading testem via `npm install testem@1.15` command `ember exam --split 3 --parallel` produces expected output:
```
ok 1 PhantomJS 2.1 - Exam Partition #3 - Acceptance | users: visiting /users
ok 2 PhantomJS 2.1 - Exam Partition #3 - ESLint | app: app.js
ok 3 PhantomJS 2.1 - Exam Partition #3 - ESLint | app: resolver.js
ok 4 PhantomJS 2.1 - Exam Partition #3 - ESLint | app: router.js
ok 5 PhantomJS 2.1 - Exam Partition #3 - ESLint | app: routes/users.js
...
```

Looks like issues is in `prependPartition` callback for `test-result` Testem event: https://github.com/trentmwillis/ember-exam/blob/master/vendor/ember-exam/test-loader.js#L35

With testem@1.15.0 it receives a test object and is able to modify it's `name` property.
With testem@1.16.0 it receives `test-result` string.

I was trying to identify what was changed in testem@1.16.0 but did not succeed: https://github.com/testem/testem/compare/v1.15.0...v1.16.0

Was I looking into wrong direction?

**Comment by trentmwillis** [comment]
Thanks for the detailed bug report. I think you're on the right path. I'm hoping to get some time to work on this addon this weekend and will try to make sure this gets addressed.

**Comment by trentmwillis** [comment]
I have PR'd a fix for this in Testem: https://github.com/testem/testem/pull/1133

**Comment by trentmwillis** [comment]
A new Testem version has been released (`v1.16.2`) which should fix this issue.
